kg4cut/.htaccess

```diff
@@ -0,0 +1,4 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^(.*)$ https://beatrizolarte.github.io/MillingTools/kg4cut_v1.2.0.ttl/$1  [R=302,L]
+RewriteRule ^$ https://beatrizolarte.github.io/MillingTools/kg4cut_v1.2.0.ttl  [R=302,L]
```

Comment:
This rule seems redundant with the previous one

Reply:
- Reversing the lines would also make this work better.  (Rules are processed in order.)
- Sub-paths of a .ttl file is a bit questionable.  Is that what you were trying to support here?